apps/kudos-react-app/src/route-helpers/route-renderer.tsx

```diff
@@ -0,0 +1,64 @@
+import { KudosRoute } from "@kudos/kudos-types";
+import {
+  RouteObject,
+  RouterProvider,
+  createBrowserRouter,
+} from "react-router-dom";
+import Dashboard from "../routes/dashboard";
+import ErrorPage from "../routes/error";
+import Home from "../routes/home";
+import Login from "../routes/login";
+import Me from "../routes/me";
+import { ProtectedRoute } from "./protected-route";
+
+export const kudosRoutes: KudosRoute[] = [
```

Comment:
This array is used to render links and routes. This could also allow us to not define a route at all if the user is not authenticated.